src/lighteval/metrics/metrics.py

```diff
@@ -123,12 +123,14 @@ class Metrics(Enum):
         corpus_level_fn=np.mean,
         higher_is_better=True,
     )
+    def compute_mean(x):
```

Comment:
Use a lambda function inside the task def

Reply:
Thanks for the review! Would you mind taking a look at the errors described here in items 3 and 4: https://github.com/huggingface/lighteval/issues/315 - I was having trouble making the lambda function work and would appreciate some advice on how to do it!

Reply:
Sure ! Can you tell me what task you are running so that I can replicate ? :)

Reply:
Thanks so much! https://github.com/chuandudx/lighteval/blob/replicate_bleurt/community_tasks/replicate_bleurt_error_eval.py
Could you try on this example task?

When I tried to run this from lighteval (as opposed to my own repo with the eval script, I ran into the formatted_doc error again). 

`TypeError: BLEURT.compute() got an unexpected keyword argument 'formatted_doc'`

I thought this was fixed by adding **kwargs in the compute function, but I wasn't able to identify what had changed. When I ran from my own repo, it worked again, so I'm unsure if there are some version updates/incompatibilities that I unfortunately was unable to get to the bottom of. 

Please let me know if this works for you though. Thanks!

Reply:
I've also observed a new error regarding saving the results that I hadn't seen before. In this case, the metrics ran fine, but had issues saving to output:
```
  File "/Users/chuandu/.pyenv/versions/3.10.6/lib/python3.10/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/chuandu/Documents/workspace/lighteval/src/lighteval/logging/evaluation_tracker.py", line 74, in default
    return super().default(o)
  File "/Users/chuandu/.pyenv/versions/3.10.6/lib/python3.10/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type float32 is not JSON serializable
```

Reply:
I just took a look this morning and your fork should work. You only need to remove this function and only call `np.mean`

Reply:
Thank you Nathan! I just implemented this change :)